charts/maildev/templates/deployment.yaml

```diff
@@ -82,9 +82,11 @@ spec:
             - name: MAILDEV_HIDE_EXTENSIONS
               value: {{ .Values.maildev.config.smtp.hideExtensions | quote }}
             {{- end }}
+            {{- if .Values.maildev.config.smtp.incoming.authRequired }}
             {{- if .Values.maildev.config.smtp.incoming.username }}
             - name: MAILDEV_INCOMING_USER
               value: {{ .Values.maildev.config.smtp.incoming.username | quote }}
+            {{- end }}
```

Comment:
So does this mean if auth is required, but no username is provided, you still need to provide the smtp-incoming-password?

Reply:
Yes. That is the assumption I'm making b'se that is how the original Chart was. I assumed there is probably a default username the application assumes if nothing is provided. I'm not sure about the application behavior, so, I didn't deviate much from the original chart.